### PR TITLE
Dynamic posts

### PR DESCRIPTION
**Pull request by baryskhanbimat**

### CODE REVIEW

File: spec/features/posts/add_post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'Add post' do
+  scenario 'Any user can add post', js: true do
+    visit posts_path
+
+    p page.body
```
Comment (yorikim) [inline spec/features/posts/add_post_spec.rb]: Лишнее

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
   def create
     @post = Post.create!(post_params)
-    redirect_to @post
+    respond_with @posts = Post.all
   end
 
-  def show
-    @post = Post.find(params[:id])
-  end
+  # def show
```
Comment (yorikim) [inline app/controllers/posts_controller.rb]: Закоментированный код не оставляй. Если тебе он понадобится в будущем, всегда можешь найти по истории коммитов.

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -1,16 +1,19 @@
 class PostsController < ApplicationController
-  def new
+  respond_to :js, :json
```
Comment (yorikim) [inline app/controllers/posts_controller.rb]: Какое действие возвращает `:json`?

File: Gemfile [hunk 4]
```diff
@@ -32,7 +32,9 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+# Responders
+# gem 'responders'
```
Comment (yorikim) [inline Gemfile]: мусор

File: .rubocop.yml [hunk 5]
```diff
@@ -126,3 +126,7 @@ Style/Documentation:
 
 Style/IndentationConsistency:
   Enabled: false
+
+Style/HashSyntax:
+  Enabled: false
```
Comment (yorikim) [inline .rubocop.yml]: Почему убрал эту проверку?

**Review by yorikim — CHANGES_REQUESTED** [review]

File: .rubocop.yml (hunk 5 above)
Comment (baryskhanbimat) [reply .rubocop.yml]: Ругался на gem 'phantomjs', :require => 'phantomjs/poltergeist'
Update your entire project to Ruby 1.9 hash syntax. А без phantomjs, capybara не пашет у меня

File: app/controllers/posts_controller.rb (hunk 3 above)
Comment (baryskhanbimat) [reply app/controllers/posts_controller.rb]: [{"id":117,"title":"1","body":"1","created_at":"2017-09-02T07:47:32.908Z","updated_at":"2017-09-02T07:47:32.908Z"},{"id":118,"title":"2","body":"2","created_at":"2017-09-02T07:47:41.117Z","updated_at":"2017-09-02T07:47:41.117Z"},{"id":119,"title":"3","body":"3","created_at":"2017-09-02T07:48:05.534Z","updated_at":"2017-09-02T07:48:05.534Z"},{"id":120,"title":"1","body":"1","created_at":"2017-09-02T07:54:04.172Z","updated_at":"2017-09-02T07:54:04.172Z"},{"id":121,"title":"2","body":"2","created_at":"2017-09-02T07:54:09.085Z","updated_at":"2017-09-02T07:54:09.085Z"},{"id":122,"title":"3","body":"3","created_at":"2017-09-02T07:54:56.485Z","updated_at":"2017-09-02T07:54:56.485Z"},{"id":123,"title":"4","body":"4","created_at":"2017-09-02T07:55:02.847Z","updated_at":"2017-09-02T07:55:02.847Z"},{"id":124,"title":"5","body":"5","created_at":"2017-09-02T07:55:08.919Z","updated_at":"2017-09-02T07:55:08.919Z"},{"id":125,"title":"12","body":"12","created_at":"2017-09-02T08:11:05.465Z","updated_at":"2017-09-02T08:11:05.465Z"}]

File: .rubocop.yml (hunk 5 above)
Comment (yorikim) [reply .rubocop.yml]: Так может лучше исправить замечание rubocop?)

File: app/controllers/posts_controller.rb (hunk 3 above)
Comment (yorikim) [reply app/controllers/posts_controller.rb]: Имею ввиду какой action возвращает json?

**Pull request merged**